### DISCUSSION TITLE
Release of v1.0

### DISCUSSION THREAD

**Discussion by pastewka** [ContactEngineering/SurfaceTopography#201] [Ideas]
Are we ready to release v1.0 @AntoineSIMTEK @mcrot - it does not seem we've had to fix anything in recent weeks.

**Comment by sannant** [comment]
I can't remember anything major. 

the reading 1D-npy files (#43) might be good to have now that we might get more users

There have been just these bugs recently, but that mainly affects power-users: 

https://github.com/ContactEngineering/SurfaceTopography/issues/184
https://github.com/ContactEngineering/SurfaceTopography/issues/185

**Comment by mcrot** [comment]
There is also still the issue https://github.com/ContactEngineering/TopoBank/issues/831 which seems to be related to SurfaceTopography and maybe fixed in 1.0.

And we still have problems with the build system, e.g. when using the package in Docker, I always get the message that installing via "wheel" failed and the old "setup.py" was used as fallback. Probably this is related: https://github.com/ContactEngineering/SurfaceTopography/issues/167   (actually assigned to me).
